assets/docs/vasudevgupta7/models/wav2vec2-960h/1.md

```diff
@@ -0,0 +1,34 @@
+# Module vasudevgupta7/wav2vec2-960h/1
+
+This model is fine-tuned on 960h of LibriSpeech dataset for Automatic Speech Recognition. For fine-tuning, base layer was initialized from pre-trained [wav2vec2](https://tfhub.dev/vasudevgupta7/wav2vec2/1)).
+
+<!-- asset-path:  https://storage.googleapis.com/gsoc-weights/wav2vec2-960h/saved-model.tar.gz -->
+<!-- task: audio-stt -->
+<!-- network-architecture: wav2vec2-960h -->
+<!-- format: saved_model_2 -->
+<!-- fine-tunable: false -->
+<!-- license: apache-2.0 -->
+<!-- language: en -->
+<!-- colab: https://colab.research.google.com/github/vasudevgupta7/gsoc-wav2vec2/blob/main/notebooks/librispeech_saved_model_evaluation.ipynb -->
+
+## Overview
+
+This model is TensorFlow equivalent of PyTorch [`facebook/wav2vec2-base-960h`](https://huggingface.co/facebook/wav2vec2-base-960h). It was published in [1].
```

Comment:
This is a little confusing. IIUC, this model was the TF equivalent of that PyTorch model, and *then* this model was fine-tuned further on LibriSpeech. Is that right? Please clarify that here if so.

Reply:
With fine-tuning runs, I am able to get WER of around `5.6% WER` (more training experiments are still in progress) while with the original pytorch checkpoint, one can get around `3% WER`. So, I decided to convert the pytorch fine-tuned checkpoint to TF using my conversion script & export that to TFHub (instead of exporting the checkpoint which I fine-tuned). This way community gets benefit of even better checkpoint (with 3% WER).

Hence this checkpoint (which I am exporting) is not fine-tuned by me but converted from PyTorch to TF. Originally, Facebook obtained this (in PyTorch) by fine-tuning their pre-trained wav2vec2 model by adding an extra LM head.

@MorganR, do you also agree here?

Thanks!

Reply:
Makes sense, thanks for clarifying.